src/main/java/core/basesyntax/model/FigureDrawing.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.model;
+
+public interface FigureDrawing {
```

Comment:
```suggestion
public interface Drawing {
```

мы можем использовать этот класс не только для фигур